db/setup.sql

```diff
@@ -1,20 +1,19 @@
-
 CREATE TABLE mods (
-    id TEXT PRIMARY KEY,
-    name TEXT,
-    developer TEXT,
+    id TEXT NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL DEFAULT 'My Mod',
+    developer TEXT NOT NULL DEFAULT 'Developer',
     download_url TEXT
 );
 
 CREATE TABLE mods_access (
-    mod_id TEXT,
-    user_id INT,
+    mod_id TEXT NOT NULL,
+    user_id INT NOT NULL DEFAULT 1,
```

Comment:
bad idea

Reply:
you mean the `DEFAULT`? or just the whole `NOT NULL` constraints in general

Reply:
the default, dont see why user id 1 should get access to everything by default, this could cause some nasty bug in the future (though not really this will all prob be replaced)

Reply:
I removed the defaults for `user_id`, even though this only would happen if someone did not provide the `user_id` field when inserting to that table.